src/vs/platform/environment/node/argvHelper.ts

```diff
@@ -17,7 +17,8 @@ function validate(args: ParsedArgs): ParsedArgs {
 	}
 
 	if (args['max-memory']) {
-		assert(args['max-memory'] >= MIN_MAX_MEMORY_SIZE_MB, `The max-memory argument cannot be specified lower than ${MIN_MAX_MEMORY_SIZE_MB} MB.`);
+		console.log(parseInt(args['max-memory']));
```

Comment:
@rebornix there is still a console.log left

Reply:
Thanks!